include/roboptim/core/function.hh

```diff
@@ -526,7 +565,6 @@ namespace roboptim
     /// \param argument point at which the function will be evaluated
     virtual void impl_compute (result_ref result, const_argument_ref argument)
       const = 0;
-
```

Comment:
The funny fact here is that I indeed used git commit -p. I should have seen it, but nope